README.md

```diff
@@ -0,0 +1,36 @@
+# rpmdeplint container image
+
+This repository contains bits needed to build a container image for the [rpmdeplint](https://pagure.io/rpmdeplint). This image can be later used by the Fedora CI.
```

Comment:
I would say "to build a container image for the [rpmdeplint] test" or simply "to build a container image for [rpmdeplint]"

Reply:
Updated, thanks ;)